test-suites/helpers/setup-env.ts

```diff
@@ -1,123 +1,9 @@
-import {step_17} from "../../deploy/tasks/deployments/testnet/steps/17_looksrare";
-import {step_18} from "../../deploy/tasks/deployments/testnet/steps/18_x2y2";
-import {step_19} from "../../deploy/tasks/deployments/testnet/steps/19_flashClaimRegistry";
-
-const buildTestEnv = async () => {
-  console.time("setup");
-  // delete json file
-  step_00();
-  console.log("------------ step 00 done ------------ ");
-
-  // deploy all mock erc20 tokens
-  await step_01();
-  console.log("------------ step 01 done ------------ ");
-
-  // deploy all mock erc721 tokens
-  await step_02();
-  console.log("------------ step 02 done ------------ ");
-
-  // deploy faucet
-  await step_03();
-  console.log("------------ step 03 done ------------ ");
-
-  // deploy PoolAddressesProvider
-  await step_04();
-  console.log("------------ step 04 done ------------ ");
-
-  // deploy ACLManager and setup ACLManager
-  await step_05();
-  console.log("------------ step 05 done ------------ ");
-
-  // deploy PoolAddressesProviderRegistry
-  await step_06();
-  console.log("------------ step 06 done ------------ ");
-
-  // deploy Pool
-  await step_07();
-  console.log("------------ step 07 done ------------ ");
-
-  // deploy PoolConfigurator
-  await step_08();
-  console.log("------------ step 08 done ------------ ");
-
-  // deploy ReservesSetupHelper
-  await step_09();
-  console.log("------------ step 09 done ------------ ");
-
-  // deploy PriceOracle and set initial prices
-  await step_10();
-  console.log("------------ step 10 done ------------ ");
-
-  // deploy mock aggregators, ParaSpaceOracle, ProtocolDataProvider, MockIncentivesController and UiPoolDataProvider
-  await step_11();
-  console.log("------------ step 11 done ------------ ");
-
-  // deploy UiIncentiveDataProviderV3
-  await step_12();
-  console.log("------------ step 12 done ------------ ");
-
-  // deploy wethGateway
-  await step_13();
-  console.log("------------ step 13 done ------------ ");
-
-  // deploy wpunkGateway
-  await step_14();
-  console.log("------------ step 14 done ------------ ");
-
-  // deploy uniswapV3Gateway
-  await step_15();
-  console.log("------------ step 15 done ------------ ");
-
-  // deploy seaport
-  await step_16();
-  console.log("------------ step 16 done ------------ ");
-
-  // deploy looksrare
-  await step_17();
-  console.log("------------ step 17 done ------------ ");
-
-  // deploy x2y2
-  await step_18();
-  console.log("------------ step 18 done ------------ ");
-
-  // deploy flash claim registry
-  await step_19();
-  console.log("------------ step 19 done ------------ ");
-
-  console.timeEnd("setup");
-};
 
 export async function testEnvFixture() {
-  await rawBRE.run("set-DRE");
-  const FORK = process.env.FORK;
-
-  if (FORK) {
-    await rawBRE.run("paraspace:mainnet");
-  } else {
-    console.log("-> Deploying test environment...");
-    await buildTestEnv();
-    printContracts();
-  }
-
+  setDRE(hre);
```

Comment:
setDRE(hre); seems not necessary because it'll be set in `deploy:all`

Reply:
Not really. `deploy:all` doesn't do that